src/components/form/importSection.svelte

```diff
@@ -1,15 +1,24 @@
 <script lang="ts">
+  import type { JsonResume } from '@src/data/JsonResume';
   import { loadData, saveResumeDataToLocalStorage } from '@src/data/data';
+  import importJsonResume from '@src/data/importJsonResume';
 
   function requestFile() {
     const fileUpload = document.getElementById('import-megaresume-data-input');
     if (fileUpload != null) fileUpload.click();
   }
 
+  function requestJsonResumeFile() {
```

Comment:
I can simplify this function to accept an id for a file input field as input, and then do the same thing. However, I may do something different with the JsonResume file import flow, like add a confirmation message.

Reply:
https://github.com/NicksPatties/megaresume/issues/77